.github/workflows/release.yml

```
@@ -42,7 +43,9 @@ jobs:
         with:
           components: rustfmt, clippy
       - run: make lint CI_MODE=1
-      - run: make test-rust TEST_THREADS=4
+      - name: Run Rust tests
+        run: make test-rust TEST_THREADS=$(nproc)
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2046:warning:1:29: Quote this to prevent word splitting

(shellcheck)